packages/calypso-e2e/src/media-helper.ts

```diff
@@ -92,3 +94,62 @@ export function getFileName( {
 export function getDateString(): string {
```

Comment:
```suggestion
export function getTimestamp(): string {
```

Reply:
I've made this change, but this also has knock-on changes in `BrowserManager` as well.